src/main/java/net/glasslauncher/mods/alwaysmoreitems/util/AlwaysMoreItems.java

```diff
@@ -5,17 +5,12 @@
 import net.glasslauncher.mods.alwaysmoreitems.recipe.ItemFilter;
 import net.glasslauncher.mods.alwaysmoreitems.registry.AMIItemRegistry;
 import net.glasslauncher.mods.alwaysmoreitems.registry.RecipeRegistry;
-import net.modificationstation.stationapi.api.mod.entrypoint.Entrypoint;
 import net.modificationstation.stationapi.api.util.Namespace;
-import net.modificationstation.stationapi.api.util.Null;
 import org.apache.logging.log4j.Logger;
 
 public class AlwaysMoreItems {
-    @Entrypoint.Namespace
-    public static final Namespace NAMESPACE = Null.get();
-
-    @Entrypoint.Logger
-    public static final Logger LOGGER = Null.get();
+    public static final Namespace NAMESPACE = Namespace.resolve();
```

Comment:
I was going to open a PR, but I see you have one up. Could you switch all of the lines edited like this to be 
```suggestion
    public static Namespace NAMESPACE = Null.get();
```

Reply:
I've been removing final to keep backwards compat with alpha.3. But, I do agree that Namespace.resolve is nicer
Doesn't hurt to move to it now for library mods like AMI. On my mods it would be a pain not to support alpha.3 when so much of the modding scene has yet to update.